gapis/api/vulkan/minimize_vertex_count.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vulkan
+
+import (
+	"context"
+
+	"github.com/google/gapid/core/log"
+	"github.com/google/gapid/core/math/u32"
+	"github.com/google/gapid/gapis/api"
+	"github.com/google/gapid/gapis/api/transform"
+)
+
+// minimizeVertexCount returns a transform that sets the maximum
+// number of vertices per draw call to 3
+func minimizeVertexCount(ctx context.Context) transform.Transformer {
+	ctx = log.Enter(ctx, "setVertexCountToOne")
+
+	const vertexCount uint32 = 3
+	const instanceCount uint32 = 1
+
+	return transform.Transform("setVertexCountToOne", func(ctx context.Context,
```

Comment:
This is no longer setting the vertex count to one.
I would also recommend setting the primitive count to one, since we may be drawing points, or lines, etc (possibly also Tessellation patches).

Reply:
Changed to using minimum number of primitives instead of vertices. Ignoring tessellation patches for now.

Reply:
So this means that anything that uses tessellation will just be disabled as far as I can tell? (Any draw calls that use tessellation will be effectively discarded).

As a follow-up we should start documenting these transforms, because this could mislead users.